kubernetes/client_factory.go

```diff
@@ -175,6 +164,31 @@ func newClientFactory(conf *kialiConfig.Config, restConfig *rest.Config) (*clien
 		f.saClientEntries[cluster] = client
 	}
 
+	if kialiConf.Clustering.IgnoreLocalCluster {
+		if len(f.saClientEntries) == 0 {
+			return nil, fmt.Errorf("kiali will exit because it has no local or remote cluster to manage. Currenly clustering.IgnoreLocalClient=true but no cluster secrets have been discovered")
```

Comment:
typo
```suggestion
			return nil, fmt.Errorf("kiali will exit because it has no local or remote cluster to manage. Currently clustering.IgnoreLocalClient=true but no cluster secrets have been discovered")
```